src/htr2hpc/train.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+import argparse
+import os
+import sys
+
+import parsl
+from parsl.configs.local_threads import config
+
+
+from htr2hpc.train_apps import prep_training_data, segtrain
+
+
+api_token_env_var = "ESCRIPTORIUM_API_TOKEN"
+
+if __name__ == "__main__":
+    try:
+        api_token = os.environ[api_token_env_var]
+    except KeyError:
+        print(
+            f"Error: eScriptorium API token must be set as environment variable {api_token_env_var}",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(
+        description="Export content from eScriptorium and train or fine-tune models"
+    )
+    parser.add_argument(
+        "base_url",
+        help="Base URL for eScriptorium instance (do not include /api/)",
+        type=str,
+    )
+    parser.add_argument(
+        "-d", "--document", help="Document id to export", type=int, dest="document_id"
+    )
+    parser.add_argument(
+        "-t", "--transcription", help="Transcription id to export", type=int
+    )
+    args = parser.parse_args()
+
+    parsl.load(config)
+    training_data = prep_training_data(
+        args.base_url,
+        api_token,
+        args.document_id,
+        args.transcription,
+    )
+
```

Comment:
Probably add logic like: if args.model exists, then run a prep_model() function taking that MODEL_PK as input and saving the model as `model_data` or something. That would handle both cases of segmentation and transcription training where the user chooses to finetune on an existing model.

To handle segmentation training "from scratch" (which is actually on top of the default blla.mlmodel), we would want logic like: if args.model does not exist and args.job == 'segmentation', then load in blla.mlmodel instead.